src/index.js

```diff
@@ -71,6 +71,13 @@ class WebSdkApi extends EventEmitter {
       throw err;
     }
 
+    try {
+      // based on discussed model, chaperone would either return null or throw an error
+      child.call("handhake")
+    } catch (err) {
+      throw err;
+    }
```

Comment:
actually, chaperone just returns an error string. I don't _think_ we can throw through postmate.

```suggestion
  const error_message = await child.call("handshake")
  if (error_message)  {
    throw new Error(error_message)
  }
```